core/src/Kokkos_Macros.hpp

```diff
@@ -245,8 +233,8 @@
 #endif
 #endif
 
-#if (1700 > KOKKOS_COMPILER_INTEL)
-#error "Compiling with Intel version earlier than 17.0 is not supported."
+#if (1900 > KOKKOS_COMPILER_INTEL)
+#error "Compiling with Intel version earlier than 19.5 is not supported."
 #endif
```

Comment:
Not sure about this. Can someone with access to 19.5 (aka 19.0.5) confirm the correct value here?